.github/workflows/test.yml

```diff
@@ -243,7 +243,8 @@ jobs:
     name: e2e
     if: github.event_name != 'schedule'
     runs-on: ubuntu-latest
-    timeout-minutes: 120
+    # 30 mins for kind, 150 mins for control-plane tests, 10 minutes for all other steps
+    timeout-minutes: 190
```

Comment:
This would be a worst case timeout that should cover: kind 30 mins + control-plane 150 minutes + give all other steps the remaining 10 minutes.

Reply:
I think this makes sense